app/views/home/dashboard.scala.html

```diff
@@ -1,58 +1,19 @@
 @import connectors.ExchangeObjects.OnlineTest
-@import models.page.DashboardPage
-@import models.page.DashboardPage._
+@import helpers.DateFormatter._
```

Comment:
Can we combine the imports above and below the view parameters declaration? Not quite sure why they are split, and appreciate it is not as a result of this commit, but it does need cleaning up